compiler/rustc_const_eval/src/interpret/operand.rs

```diff
@@ -645,6 +648,16 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
         }
     }
 
+    /// Tries to evaluate an unevaluated constant from the MIR (and not the type-system).
```

Comment:
I don't understand this comment. The type of the argument is `ty::Unevaluated`, that sounds like a type system thing to me?